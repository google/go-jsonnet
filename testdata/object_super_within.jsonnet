{"a": 42, d: 1} + {c: super.d, d: 42} + {"a": 2, b: super.c, d: 42}