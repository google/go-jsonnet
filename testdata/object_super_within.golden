{
   "a": 2,
   "b": 1,
   "c": 1,
   "d": 42
}
